packages/@o3r/localization/schematics/ng-add/index.ts

```diff
@@ -72,6 +72,9 @@ function ngAddFn(options: NgAddSchematicsSchema): Rule {
       };
     });
     const registerDevtoolRule = await registerDevtools(options);
+    const schematicDefaultOptions = Object.fromEntries(['@o3r/core:component*'].map((name) => [name, {
```

Comment:
are we really expecting more entries? or is it just a trick to make the linter happy?

Reply:
For some schematics we have more entries 
So I did it to have the same format everywhere I saw it